packages/babel-types/lib/builders/generated/deprecated.d.ts

```diff
@@ -0,0 +1,2 @@
+// exports with old naming are not expected to be used from typescript
```

Comment:
Why is this file committed?

Reply:
there is `src/builders/generated/deprecated.d.ts` but typescript does not output `d.ts` files from other `d.ts` files, this a workaround to add the file into lib folder